lib/govuk_app_config/govuk_logging.rb

```diff
@@ -1,32 +1,31 @@
+require "json"
 require "logstasher"
 require "action_controller"
-require_relative "rails_ext/action_dispatch/debug_exceptions"
 
 module GovukLogging
   def self.configure
-    # GOV.UK Rails applications are expected to output JSON to stdout which is
-    # then indexed in a Kibana instance. These log outputs are created by the
-    # logstasher gem.
-    #
-    # Rails applications will typically write other things to stdout such as
-    # `Rails.logger` calls or 'puts' statements. However these are not in a
-    # JSON format which causes problems for the log file parsers.
-    #
-    # To resolve this we redirect stdout to stderr, to cover any Rails
-    # writing. This frees up the normal stdout for the logstasher logs.
-    #
     # We also disable buffering, so that logs aren't lost on crash or delayed
```

Comment:
```suggestion
    # We disable buffering, so that logs aren't lost on crash or delayed
```

Given we've lost the above comments the "also" doesn't fit.